### PR TITLE
BD-2162: Phone number capture template 

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-2162**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (releasing later, documentation being used for QA party)
- [ ] No

### CODE REVIEW

File: _docs/_hidden/private_betas/phone_number_capture.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+nav_title: Phone Number Capture
+article_title: Phone Number Capture
+permalink: "/phone_number_capture/"
+description: "This reference page covers how to create a phone number capture form with the in-app message drag & drop editor."
+---
+
+# Phone number capture form
+
+> Learn how to create an in-app message to collect users' phone numbers capture  form.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/phone_number_capture.md:10]:
```suggestion
> Learn how to create an in-app message to collect users' phone numbers capture form.
```

Comment (lydia-xie) [inline _docs/_hidden/private_betas/phone_number_capture.md:10]:
```suggestion
> Learn how to create an in-app message to collect users' phone numbers capture form.
```

File: _docs/_hidden/private_betas/phone_number_capture.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+The phone number capture form is a template available in the Drag & Drop Editor for in-app messages. Use this template to collect users' phone numbers and grow your subscription groups.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/phone_number_capture.md:12]:
```suggestion
The phone number capture form is a template available in the drag-and-drop editor for in-app messages. Use this template to collect users' phone numbers and grow your subscription groups.
```
I believe "Drag & Drop Editor" in the UI has been updated to be more "Drag-And-Drop Editor" so I think we can refer to it as "drag-and-drop editor" in general... 🤔

File: _docs/_hidden/private_betas/phone_number_capture.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+![][img1]{: style="max-width:35%"}
+
+## SDK requirements
+
+In addition to the [minimum SDK versions][1] required to receive messages built using the Drag & Drop Editor, users will need to be on the following minimum SDK versions to include text links that do not dismiss the message:
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/phone_number_capture.md:18]:
```suggestion
In addition to the [minimum SDK versions][1] required to receive messages built using the drag-and-drop editor, users will need to be on the following minimum SDK versions to include text links that do not dismiss the message:
```
same note/comment

File: _docs/_hidden/private_betas/phone_number_capture.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+nav_title: Phone Number Capture
+article_title: Phone Number Capture
+permalink: "/phone_number_capture/"
+description: "This reference page covers how to create a phone number capture form with the in-app message drag & drop editor."
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/phone_number_capture.md:5]:
```suggestion
description: "This reference page covers how to create a phone number capture form with the in-app message drag-and-drop editor."
```

File: _docs/_hidden/private_betas/phone_number_capture.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+{% alert note %}
+This documentation is not intended to provide, nor may it be relied fully upon, as providing legal advise.
+{% endalert %}
+
+For more information about SMS compliance, see [SMS Laws and Regulations][4].
+
+### Step 4: Style your message
+
+You can customize the look and feel of your message using the Drag & Drop Editor [in-app message components][3].
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/phone_number_capture.md:78]:
```suggestion
You can customize the look and feel of your message using the drag-and-drop [in-app message components][3].
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
lgtm - left some comments for your review

File: _docs/_hidden/private_betas/phone_number_capture.md (hunk 2 above)
Comment (bre-fitzgerald) [reply _docs/_hidden/private_betas/phone_number_capture.md:12]: oop good catch, ty!

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**